File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocso/OCS1003Q02GridOUT1001Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocso
{
    [Serializable]
	public class OCS1003Q02GridOUT1001Info
	{
		private String _jubsu;
		private String _reserYn;
		private String _jinryoTime;
		private String _gwaName;
		private String _doctorName;
		private String _naewonYnName;
		private String _bunho;
		private String _naewonDate;
		private String _gwa;
		private String _doctor;
		private String _naewonType;
		private String _jubsuNo;
		private String _pkNaewon;
		private String _orderEndYn;

		public String Jubsu
		{
			get { return this._jubsu; }
			set { this._jubsu = value; }
		}

		public String ReserYn
		{
			get { return this._reserYn; }
			set { this._reserYn = value; }
		}

		public String JinryoTime
		{
			get { return this._jinryoTime; }
			set { this._jinryoTime = value; }
		}

		public String GwaName
		{
			get { return this._gwaName; }
			set { this._gwaName = value; }
		}

		public String DoctorName
		{
			get { return this._doctorName; }
			set { this._doctorName = value; }
		}

		public String NaewonYnName
		{
			get { return this._naewonYnName; }
			set { this._naewonYnName = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String NaewonDate
		{
			get { return this._naewonDate; }
			set { this._naewonDate = value; }
		}

		public String Gwa
		{
			get { return this._gwa; }
			set { this._gwa = value; }
		}

		public String Doctor
		{
			get { return this._doctor; }
			set { this._doctor = value; }
		}

		public String NaewonType
		{
			get { return this._naewonType; }
			set { this._naewonType = value; }
		}

		public String JubsuNo
		{
			get { return this._jubsuNo; }
			set { this._jubsuNo = value; }
		}

		public String PkNaewon
		{
			get { return this._pkNaewon; }
			set { this._pkNaewon = value; }
		}

		public String OrderEndYn
		{
			get { return this._orderEndYn; }
			set { this._orderEndYn = value; }
		}

		public OCS1003Q02GridOUT1001Info() { }

		public OCS1003Q02GridOUT1001Info(String jubsu, String reserYn, String jinryoTime, String gwaName, String doctorName, String naewonYnName, String bunho, String naewonDate, String gwa, String doctor, String naewonType, String jubsuNo, String pkNaewon, String orderEndYn)
		{
			this._jubsu = jubsu;
			this._reserYn = reserYn;
			this._jinryoTime = jinryoTime;
			this._gwaName = gwaName;
			this._doctorName = doctorName;
			this._naewonYnName = naewonYnName;
			this._bunho = bunho;
			this._naewonDate = naewonDate;
			this._gwa = gwa;
			this._doctor = doctor;
			this._naewonType = naewonType;
			this._jubsuNo = jubsuNo;
			this._pkNaewon = pkNaewon;
			this._orderEndYn = orderEndYn;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocso/OCS1003Q02GridOUT1001Info.java
package nta.med.data.model.ihis.ocso;

import java.util.Date;

public class OCS1003Q02GridOUT1001Info {
    private String jubsu ;
	private String reserYn ;
	private String jinryoTime ;
	private String gwaName ;
	private String doctorName ;
	private String naewonYnName ;
	private String bunho ;
	private Date naewonDate;
	private String gwa ;
	private String doctor ;
	private String naewonType ;
	private Double jubsuNo ;
	private Double pkNaewon ;
	private String orderEndYn ;
	public OCS1003Q02GridOUT1001Info(String jubsu, String reserYn,
			String jinryoTime, String gwaName, String doctorName,
			String naewonYnName, String bunho, Date naewonDate, String gwa,
			String doctor, String naewonType, Double jubsuNo, Double pkNaewon,
			String orderEndYn) {
		super();
		this.jubsu = jubsu;
		this.reserYn = reserYn;
		this.jinryoTime = jinryoTime;
		this.gwaName = gwaName;
		this.doctorName = doctorName;
		this.naewonYnName = naewonYnName;
		this.bunho = bunho;
		this.naewonDate = naewonDate;
		this.gwa = gwa;
		this.doctor = doctor;
		this.naewonType = naewonType;
		this.jubsuNo = jubsuNo;
		this.pkNaewon = pkNaewon;
		this.orderEndYn = orderEndYn;
	}
	public String getJubsu() {
		return jubsu;
	}
	public void setJubsu(String jubsu) {
		this.jubsu = jubsu;
	}
	public String getReserYn() {
		return reserYn;
	}
	public void setReserYn(String reserYn) {
		this.reserYn = reserYn;
	}
	public String getJinryoTime() {
		return jinryoTime;
	}
	public void setJinryoTime(String jinryoTime) {
		this.jinryoTime = jinryoTime;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getNaewonYnName() {
		return naewonYnName;
	}
	public void setNaewonYnName(String naewonYnName) {
		this.naewonYnName = naewonYnName;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public Date getNaewonDate() {
		return naewonDate;
	}
	public void setNaewonDate(Date naewonDate) {
		this.naewonDate = naewonDate;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public String getNaewonType() {
		return naewonType;
	}
	public void setNaewonType(String naewonType) {
		this.naewonType = naewonType;
	}
	public Double getJubsuNo() {
		return jubsuNo;
	}
	public void setJubsuNo(Double jubsuNo) {
		this.jubsuNo = jubsuNo;
	}
	public Double getPkNaewon() {
		return pkNaewon;
	}
	public void setPkNaewon(Double pkNaewon) {
		this.pkNaewon = pkNaewon;
	}
	public String getOrderEndYn() {
		return orderEndYn;
	}
	public void setOrderEndYn(String orderEndYn) {
		this.orderEndYn = orderEndYn;
	}
	
}
